kokoro/gcp-ubuntu/kokoro_build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
All source files need to have a license header. Please add them.

Example: https://github.com/google/dwh-assessment-extraction-tool/blob/main/src/scripts/run-td-extract.sh